src/usage_metrics/core/s3.py

```diff
@@ -52,15 +23,43 @@ def core_s3_logs(
 
     Add column headers, geocode values,
     """
+    # Name columns
+    raw_s3_logs.columns = [
```

Comment:
It'll be so annoying if this order changes but I don't have a good way around that.